tests/code_update.py

```diff
@@ -51,6 +51,180 @@ def test_verify_quotes(network, args):
 
     return network
 
+@reqs.description("Node with correct security policy joins successfully")
+def test_add_node_with_correct_security_policy(network, args):
+
+    if not IS_SNP:
+        LOG.warning("Skipping test_add_node_with_correct_security_policy with non SNP platform")
+        return network
+
+    replacement_package = (
+        "samples/apps/logging/liblogging"
+        if args.package == "libjs_generic"
+        else "libjs_generic"
+    )
+
+    primary, others = network.find_nodes()
+    prev_nodes = [primary, *others]
+    new_node = network.create_node("local://localhost")
+    network.join_node(new_node, replacement_package, args, timeout=3)
+    network.trust_node(new_node, args)
+    primary, others = network.find_nodes()
+    nodes = [primary, *others]
+    assert len(nodes) == len(prev_nodes) + 1, "New node not added to network"
+
+    with primary.client() as client:
+        r = client.get("/gov/security_policy")
+        policies = sorted(r.body.json()["policies"], key=lambda x: x["digest"])
+
+    LOG.info(f"Checking governance table security policies")
+    expected = [
+        {
+            "digest": DEFAULT_SNP_SECURITY_POLICY_DIGEST,
+            "raw": DEFAULT_SNP_SECURITY_POLICY,
+        }
+    ]
+    expected.sort(key=lambda x: x["digest"])
+    assert policies == expected, [(a, b) for a, b in zip(policies, expected)]
+
+@reqs.description("Node with no security policy set but good digest matching ledger joins successfully")
+def test_add_node_with_no_raw_security_policy_match_ledger(network, args):
+
+    if not IS_SNP:
+        LOG.warning("Skipping test_add_node_with_correct_security_policy with non SNP platform")
+        return network
+
+    replacement_package = (
```

Comment:
I don't think we need to use the replacement package here and simply `join_node(new_node, args.package)` for conciseness?

Reply:
Haha yeah I just did this, great minds